src/main/java/org/jenkinsci/plugins/terraform/TerraformBuildWrapper.java

```diff
@@ -87,25 +88,27 @@ public String getUrlName() {
     private final boolean doNotApply;
     private final Configuration config;
     private final String terraformInstallation;
+    private final String workDir;
     private FilePath stateFile;
     private FilePath configFile;
     private FilePath variablesFile;
     private FilePath workspacePath;
     private FilePath workingDirectory;
 
-    private static final String WORK_DIR_NAME = "terraform-plugin";
+    //private static final String WORK_DIR_NAME = "terraform-plugin";
```

Comment:
this should be left as a default, as without setting an explicit directory for workDir, files are written to your workspace